src/ExecClient.cs

```diff
@@ -9,6 +9,10 @@ namespace k8s
 {
     public class ExecClient : IDisposable
     {
+        private const int StdInStreamIndex = 0;
```

Comment:
const from py client
https://github.com/kubernetes-client/python-base/blob/master/stream/ws_client.py#L24

Reply:
Thx, I added the other channel indices as well.